src/common/LogEntry.h

```diff
@@ -97,24 +105,22 @@ WRITE_CLASS_ENCODER_FEATURES(LogEntry)
 struct LogSummary {
   version_t version;
   list<LogEntry> tail;
+  ceph::unordered_set<LogEntryKey> keys;
```

Comment:
if we want to be spacial efficient, probably we can just keep the hashed `LogEntryKey` around?